3.4/Dockerfile.rhel7

```diff
@@ -35,9 +35,10 @@ LABEL summary="$SUMMARY" \
 # To use subscription inside container yum command has to be run first (before yum-config-manager)
 # https://access.redhat.com/solutions/1443553
 RUN yum repolist > /dev/null && \
+    yum-config-manager --disable \* &> /dev/null && \
+    yum-config-manager --enable rhel-7-server-rpms && \
     yum-config-manager --enable rhel-server-rhscl-7-rpms && \
```

Comment:
In the version 3.4 the ordering of the repositories is difeerent than in 2.7 and 3.5. It might be worth fixing that so the files are as similar as possible.